env0/resource_module.go

```diff
@@ -99,10 +99,6 @@ func resourceModuleCreate(ctx context.Context, d *schema.ResourceData, meta inte
 		return diag.Errorf("schema resource data deserialization failed: %v", err)
 	}
 
-	if len(payload.TokenId) > 0 {
```

Comment:
for modules I decided to remove the implicit IsGitlab (This is less backward compatible).